packages/orbit-design-tokens/src/index.ts

```diff
@@ -11,4 +13,4 @@ export { default as tokensToCssVars } from "./tokensToCssVars";
 export { default as defaultFoundation } from "./js/defaultFoundation";
 export { default as convertHexToRgba } from "./js/convertHexToRgba";
 export { default as convertRgbaToHex } from "./js/convertRgbaToHex";
-export { createTheme, PaletteColors };
+export { createTheme, PaletteColors, parseToRgba };
```

Comment:
Sorry didn't notice it before. But now we don't need to export this, right?

Reply:
right 👍🏻